pyproject.toml

```diff
@@ -69,7 +69,7 @@ version = "1.1.0"
 description = "A web service for scanning media hosted by a Matrix media repository"
 authors = ["Element Backend Team <team-backend-synapse@element.io>"]
 readme = "README.md"
-license = "AGPL-3.0-only"
+license = "AGPL-3.0-only OR LicenseRef-Element-Commercial"
```

Comment:
Could you put this in a separate PR? It seems unrelated to the doc change.

Reply:
Moved here: https://github.com/element-hq/matrix-content-scanner-python/pull/71